apis/filters/parse_test.go

```diff
@@ -223,3 +223,54 @@ func TestArgsMatchKVList(t *testing.T) {
 		}
 	}
 }
+
+func TestArgsValidate(t *testing.T) {
+	tests := []struct {
+		name     string
+		testArgs Args
+		accepted map[string]bool
+		wantErr  bool
+	}{
+		{
+			"mapping keys are in the accepted set",
```

Comment:
@WillSmisi since there are many fields for the test case, could we add the filed name when we do assignment? something like that
```
{
  name: xxx,
  testArgs: yyy,
  accepted: zzz,
 ...
}
```
It can be more clear for other. Thanks